libsolidity/formal/SMTChecker.cpp

```diff
@@ -909,3 +894,16 @@ void SMTChecker::addPathImpliedExpression(smt::Expression const& _e)
 {
 	m_interface->addAssertion(smt::Expression::implies(currentPathConditions(), _e));
 }
+
+void SMTChecker::clearLocalVariables()
```

Comment:
`clear` sounds like they are set to `unknown`. What about `remove` or something similar?

Reply:
`remove` sounds good